server/data/redirects.json

```diff
@@ -6,6 +6,9 @@
   "/docs/api/components/nav/NavController": "/docs/api/navigation/NavController",
   "/docs/api/menu/Menu": "/docs/api/components/menu/Menu",
   "/docs/api/tab/Tab": "/docs/api/components/tabs/Tab",
+  "/docs/cli/build": "/docs/cli",
```

Comment:
Wouldn't it be better to redirect to the `ionic cordova *` commands instead of /?